src/components/Article.jsx

```diff
@@ -24,17 +29,41 @@ const Article = () => {
           <h1 data-cy='article-title' className='article-title'>
             {article.title}
           </h1>
+          <p data-cy='article-category' className='article-category'>
+            Category: {article.category}
+          </p>
+          <Popup
+            content={
+              popupSuccess
+                ? 'Thank you for your opinion!'
+                : 'Sorry your vote was not registered'
+            }
+            on='Click'
+            pinned
+            popper={{id: 'rating-message'}}
+            hideOnScroll
+            trigger={
+              <Rating
+                onRate={articleRating}
+                data-cy='article-rating-button'
+                defaultRating={Math.floor(article.rating)}
```

Comment:
with floor function, it will never be 5 maybe it is better to average it so 4,6 will be 5. Also does it take only integers? Have you tried to put a number with decimal in it?

Reply:
Math.floor / Math.round will take the float and turn it into an integer. The Rating element doesn't like decimal numbers. 
We can discuss the rounding part. 
If it has straight 5 stars it'll be a 5, no? :P

Reply:
yes, but hardly anything has pure 5 star rating. 100 people voted 5, one 4, and element will display 4 starts, not very good I think.